package.json

```diff
@@ -47,7 +47,6 @@
         "eslint": "^4.13.1",
         "eslint-config-prettier": "^2.9.0",
         "handlebars": "^4.0.11",
-        "husky": "^0.14.3",
```

Comment:
Do we not use Husky for commitlint?

https://github.com/marionebl/commitlint#getting-started
https://github.com/typicode/husky/blob/master/HOOKS.md#hooks

Reply:
![giphy 21](https://user-images.githubusercontent.com/1307267/36731513-53298258-1bcb-11e8-8762-7d9aee5c58a0.gif)
Will fix!

Reply:
Good catch. Yes, we need husky for commitlint.
https://github.com/SpareBank1/designsystem/blob/5a01f4abf7a5e88c214d7bc7ccacdafe889e0c88/package.json#L14